tools/osbuilder/rootfs-builder/nvidia/nvidia_rootfs.sh

```diff
@@ -15,6 +15,7 @@ readonly SCRIPT_DIR="${script_dir}/nvidia"
 # This will control how much output the inird/image will produce
 DEBUG=""
 KBUILD_SIGN_PIN=${KBUILD_SIGN_PIN:-}
+AGENT_POLICY=${AGENT_POLICY:-}
```

Comment:
@Champ-Goblem, just for the sake of consistency with what we have as part of the other rootfs scripts ...
Could you sete the default policy here to "no"?

```suggestion
AGENT_POLICY="${AGENT_POLICY:-no}"
```

Reply:
I've fixed this and force-pushed to your branch, @Champ-Goblem!